tests/unittests/Caffe2ImporterTest.cpp

```diff
@@ -844,17 +836,20 @@ TEST(caffe2, importClipDefault) {
     ctx.allocate(mod.getPlaceholders());
     updateInputPlaceholdersByName(ctx, &mod, {"inputs_0"}, {&inputs_0});
   }
-
-  auto *res = ctx.get(output);
-  EE.compile(CompilationMode::Infer, F);
-  EE.run(ctx);
-
-  auto result = res->getHandle();
-  std::vector<size_t> expectedDims = {5, 5};
-  EXPECT_TRUE(result.dims().vec() == expectedDims);
-  for (size_t i = 0; i < 5 * 5; i++) {
-    EXPECT_FLOAT_EQ(result.raw(i), inputs_0.getHandle().raw(i));
-  }
+  EXPECT_EQ(F->getNodes().size(), 5);
+  auto *saveNode = getSaveNodeFromDest(output);
+  auto *minNode = llvm::dyn_cast<MinNode>(saveNode->getInput().getNode());
+  ASSERT_TRUE(minNode);
+  auto *maxNode = llvm::dyn_cast<MaxNode>(minNode->getLHS().getNode());
+  ASSERT_TRUE(maxNode);
+  auto *maxSplatNode = llvm::dyn_cast<SplatNode>(minNode->getRHS().getNode());
```

Comment:
Can you also check that the input to this node is the expected default value? You can access it using `maxSplatNode->getValue()`.